zeebe.js

```diff
@@ -0,0 +1,96 @@
+import axios from "axios";
+import { getAccessToken } from "./auth.js";
+
+const authorizationConfiguration = {
+  clientId: process.env.ZEEBE_CLIENT_ID,
+  clientSecret: process.env.ZEEBE_CLIENT_SECRET,
+  audience: process.env.ZEEBE_AUDIENCE
+};
+
+// An action that assigns a user task.
+async function assignUser([assignee]) {
```

Comment:
```suggestion
async function assignUser([userTaskKey, assignee]) {
```

And then associated, line 17 would be deleted.

### explanation

I'm not sure if I'd block merge for this or not, but I feel pretty strongly that there should be two arguments into this function -- the `userTaskKey`, and the `assignee`. Let's talk about it and make a decision together 😅 

Currently, this method reads the `userTaskKey` from an environment variable (line 17 below). I don't think that's the correct approach for a CLI of this nature. 

Things that are read from the environment variables should not change frequently during the use of the CLI. We've read variables from the environment for previous API tutorials, like the cluster ID in the administration API -- however that is a case where the cluster ID is unlikely to vary in usage of the CLI. A person likely has one cluster they want to interact with, most if not all of the time. 

In this case, every time I call this endpoint I'm likely to use a different `userTaskKey`. To do that, as a developer I'd much prefer to pass it in as an argument, e.g. `npm run cli zeebe assign 123455-task-id-1234324 assignee@assignee.com`, than to change the environment variables. 

I can be talked out of this being blocking, because this isn't _really_ a production-grade CLI we're building. We're trying to demonstrate interactions with an API. The design of the CLI is really up to the user if they want to construct it differently, and maybe we don't care. On the other hand, I do think it's nice to build something that mimics a production-like example, and in this case, I don't think a production example of this CLI would require the user task ID to be changed in the environment file every time.

Reply:
> On the other hand, I do think it's nice to build something that mimics a production-like example

Based on user feedback, this tends to be the largest request when it comes to all examples across docs, so I would consider this blocking too!

> To do that, as a developer I'd much prefer to pass it in as an argument, e.g. npm run cli zeebe assign 123455-task-id-1234324 assignee@assignee.com, than to change the environment variables.

I agree with this solution! I'll adjust in the docs, and request review from you and @akeller (as @pepopowitz is out of FTO 😄)